test/integration/users.test.js

```diff
@@ -144,6 +144,47 @@ describe("Users", function () {
           });
       });
     });
+
+    it("Should not remove old roles when updating user roles", async function () {
+      let newUserId;
+      await addUser(newUser).then((userId) => {
+        newUserId = userId;
+      });
```

Comment:
Nit : If we are using async await, it is better to avoid chaining .then() promises, as this can lead to code that is difficult to read and maintain.

Reply:
Thanks for pointing out I have removed